strings/brain_damage_lines.json

```diff
@@ -61,7 +61,8 @@
         "@pick(semicolon)N-NYAAAAAA~",
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
-        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)"
+        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
+		"@pick(semicolon)okay i'm reporting shitcurity don't grab prisoners with you is 1 shitcurity is 2 and asscurity is 3 = permaban i'm reporting"
```

Comment:
that's a "this is a game" thing, not a "admins be spying on yo ass" thing.
plus we had that whole thing with the "bans" in descriptions of items.